notebooks/community/ml_ops/stage6/get_started_with_automl_tabular_model_online.ipynb

```diff
@@ -0,0 +1,1154 @@
+{
```

Comment:
<p>Do we use <code>Tutorial</code> title in our notebooks? Or is it a title for get_started notebooks? Otherwise I'd change it to <code>Training AutoML Model</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/961/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
It's leftover from an older style notebook. Will change the title